src/exercises/1-2-Design_Tokens/final/Button.js

```diff
@@ -6,7 +6,7 @@ const Button = styled.button`
 	color: ${props => props.theme.colors.background};
 	background-color: ${props => props.theme.colors.primary};
 	border: 0;
-	border-radius: ${props => props.theme.radii.round};
+	border-radius: ${props => props.theme.radii.round}px;
```

Comment:
This probably should be in the theme itself? All other values are hardcoded as rems in the theme file.

Reply:
Done